typescript/infra/src/govern/HyperlaneAppGovernor.ts

```diff
@@ -132,21 +130,6 @@ export abstract class HyperlaneAppGovernor<
 
   protected abstract mapViolationsToCalls(): Promise<void>;
 
-  handleProxyViolation(violation: ProxyViolation) {
```

Comment:
we still need this no?

Reply:
I thought we agreed that this would remove upgrade functionality?